tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
```

Comment:
I don't think we need these ts and tsx
```suggestion
    "./src/**/*.{js,jsx}",
  ],
```